### PR TITLE
imgur cheatsheet created

### PR DESCRIPTION
**Pull request by RomainLG**
OK let's hope that this is the first of a long streak of instant answers!

Thanks @moollaza for the help :)

-------------

# Goodie Pull Request Template

Using this template will help us better understand your Instant Answer and assist you when necessary.  Simply copy and paste the markdown below into the description of your GitHub pull request and complete as appropriate.

**What does your Instant Answer do?**

Provides shortcuts for navigating Imgur.com

**What problem does your Instant Answer solve (Why is it better than organic links)?**

It's simpler than to find the relevant infos (no official source for keyboard shortcuts)

**What is the data source for your Instant Answer? (Provide a link if possible)**

Something like https://imgur.com/gallery/nz63EPD but it's not official nor up to date.

**Why did you choose this data source?**

**Are there any other alternative (better) data sources?**

**What are some example queries that trigger this Instant Answer?**

imgur cheatsheet

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**

Imgurians!

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**

**Which existing Instant Answers will this one supercede/overlap with?**

**Are you having any problems? Do you need our help with anything?**

**Where did you hear about DuckDuckHack? (For first time contributors)**

From the blog!

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**

## Checklist
Please place an 'X' where appropriate.

```
- [x] Added metadata and attribution information (https://duck.co/duckduckhack/metadata)
- [] Wrote test file and added to t/ directory (https://duck.co/duckduckhack/goodie_tests)
- [x] Verified that instant answer adheres to design guidelines (https://duck.co/duckduckhack/design_styleguide)
- [x] Verified that instant answer adheres to code styleguide (https://duck.co/duckduckhack/code_styleguide)
- [] Tested cross-browser compatibility

    Please let us know which browsers/devices you've tested on:
    - Windows 8
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 10

    - Windows 7
        - [x] Google Chrome
        - [] Firefox
        - [] Opera
        - [] IE 8
        - [] IE 9
        - [] IE 10

    - Windows XP
        - [] IE 7
        - [] IE 8

    - Mac OSX
        - [] Google Chrome
        - [] Firefox
        - [] Opera
        - [] Safari

    - iOS (iPhone)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - iOS (iPad)
        - [] Safari Mobile
        - [] Google Chrome
        - [] Opera

    - Android
        - [] Firefox
        - [] Native Browser
        - [] Google Chrome
        - [] Opera

```
image

-----
IA Page: https://duck.co/ia/view/imgur_cheat_sheet

### CODE REVIEW

**Comment by moollaza** [comment]
@RomainLG thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

>OK let's hope that this is the first of a long streak of instant answers!
>Thanks @moollaza for the help :)

Not a problem! Always happy to help, and definitely excited to see more Instant Answers :smile:

File: share/goodie/cheat_sheets/imgur.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+    "id": "imgur_cheat_sheet",
+    "name": "imgur",
+    "description": "Imgur shortcuts",
+    "metadata": {
+        "sourceName": "ImgurCheatSheet",
+        "sourceUrl": "https://i.imgur.com/nz63EPD.png",
+        "category": "cheat_sheets",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/imgur.json]: Please remove these, this info is actually handled by the IA Page: https://duck.co/ia/view/imgur_cheat_sheet (will be generated soon...)

If you’d like to have edit permission for your IA Page(s) please make a duck.co account, then link your GitHub account to it, and finally send me your duck.co account name and a list of the in-progress IA’s you’re working on. I'll get the team to give you edit permissions so you can fill out the IA pages.

File: share/goodie/cheat_sheets/imgur.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    "section_order": [
+        "Voting",
+        "Accessibility",
+        "On the Homepage"
+    ],
+    "sections": {
+        "Voting": [{
+            "val": "Favorite",
+            "key": "[Inser] or [0]"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/imgur.json]: Looks like you're missing the `t` in "Insert"

**Comment by moollaza** [comment]
@RomainLG don't forget to test this and add a screenshot to the PR template above!

File: share/goodie/cheat_sheets/imgur.json (hunk 1 above)
Comment (RomainLG) [inline share/goodie/cheat_sheets/imgur.json]: Dammit! [This page](https://duck.co/duckduckhack/metadata) asks for metadata, attribution and all to be added.

I'm removing them, I'll know for the next time. 

My Duck.co account is RomainLG.

As for the list of WIP IA I haven't decided what's next yet.

File: share/goodie/cheat_sheets/imgur.json (hunk 2 above)
Comment (RomainLG) [inline share/goodie/cheat_sheets/imgur.json]: My keyboard doesn't have the `t`, it says `Inser` but if you'd prefer I add it...

**Comment by RomainLG** [comment]
Hi @moollaza, I've got a question...

My master branch was behind the ddg master, I had no luck merging with it.
I decided to `git pull duckduckgo/zeroclickinfo-goodies` to my codio and then `git push` this to `https://github.com/RomainLG/zeroclickinfo-goodies`.

But what will happen to this PR? It'll be deleted I guess wouldn't it?

**Comment by moollaza** [comment]
>EDIT2: I'm OK now, but that took the day.

@RomainLG sorry about the delay and all the trouble -- I'm glad you have it working now! I hope you learned something? Hehe Git can be a pain at first, but once you start to understand it, everything becomes much easier and more clear :+1:

File: share/goodie/cheat_sheets/imgur.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
+{
+    "id": "imgur_cheat_sheet",
+    "name": "imgur",
+    "description": "Imgur shortcuts",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/imgur.json]: Let's remove this -- it's a bit redundant :)

File: share/goodie/cheat_sheets/imgur.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    "metadata": {
+        "sourceName": "Imgur Cheatsheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/imgur.json]: Let's remove "cheatsheet" from the source. "Imgur" is fine :+1:

File: share/goodie/cheat_sheets/imgur.json [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        "sourceUrl": "https://i.imgur.com/nz63EPD.png"
+    },
+    "section_order": [
+        "Voting",
+        "Accessibility",
+        "On the Homepage"
+    ],
+    "sections": {
+        "Voting": [{
+            "val": "Favorite",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/imgur.json]: This one doesn't quite display correctly right now (the parser is a little strict!)

Lets do it this way:

```
{
   "val": "Favourite",
   "key": "0"
}, {
   "val": "Favourite (alternative)",
   "key": "0"
 }
```

File: share/goodie/cheat_sheets/imgur.json [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+            "key": "([0]/[Insert])"
+        }, {
+            "val": "Upvote",
+            "key": "+"
+        }, {
+            "val": "Downvote",
+            "key": "-"
+        }],
+        "Accessibility":[{
+            "val": "Previous / Next image",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/imgur.json]: Likewise, let's break this up:

This one doesn't quite display correctly right now (the parser is a little strict!)

Lets do it this way:

```
{
   "val": "Previous Image",
   "key": "←"
}, {
   "val": "Next Image",
   "key": "→"
 }
```

File: share/goodie/cheat_sheets/imgur.json (hunk 3 above)
Comment (RomainLG) [inline share/goodie/cheat_sheets/imgur.json]: You mean remove this whole part? Starting with metadata?

**Comment by moollaza** [comment]
@RomainLG I've continued and merged this IA in #1570. There were some outstanding issues but I figured it would simple enough for me to help you out with them this time around.

Thanks for taking the time to contribute! We really appreciate it.

We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Pull request closed without merging**